jenkins/spark-tests.sh

```diff
@@ -195,9 +195,6 @@ run_iceberg_tests() {
 
   # Iceberg does not support Spark 3.3+ yet
   if [[ "$ICEBERG_SPARK_VER" < "3.3" ]]; then
-    # Classloader config is here to work around classloader issues with
```

Comment:
Why is this removed as a part of this PR?

Reply:
it should have been removed in #5646 because the comment refers to force.caller.classloader=false that no longer exists.